neuron/src/lib/Neuron/Web/Query/View.hs

```diff
@@ -98,11 +98,11 @@ renderQuery someQ =
     case someQ of
       Some (ZettelQuery_ZettelByID _ _) ->
         blank
-      Some (ZettelQuery_ZettelsByTag [] _mconn _mview) ->
+      Some (ZettelQuery_ZettelsByTag [] _maybeLimit _mconn _mview) ->
         text "All zettels"
-      Some (ZettelQuery_ZettelsByTag (fmap unTagPattern -> pats) _mconn _mview) -> do
+      Some (ZettelQuery_ZettelsByTag (fmap unTagPattern -> pats) maybeLimit _mconn _mview) -> do
         let qs = toText $ intercalate ", " pats
-            desc = toText $ "Zettels tagged '" <> qs <> "'"
+            desc = toText $ (maybe "" (\limit -> show limit <> " ") maybeLimit) <> "Zettels tagged '" <> qs <> "'"
```

Comment:
My main problem with this is that I'm not much of a UI designer. I like the idea, but don't know how to go about it yet. I'll think about it and comment my ideas here.

Reply:
How would I go about calculating `M` here? I have the query result in `renderQueryResult` and can count the contained zettels, but that is already the limited amount. To get `M` I would have to recalculate the query without the limit, wouldn't I?

If, as you say below, the limit is supposed to *only* affect the view, it would make sense to apply the limit only in `renderQueryResult` and not before. This would make it easier to calculate the `M`. However, as I said below, I think the limit should affect the graph.

What are your thoughts on this?